src/middlewared/middlewared/service.py

```diff
@@ -183,7 +205,8 @@ class SystemServiceService(ConfigService):
                                    {'prefix': self._config.datastore_prefix})
 
         enabled = (await self.middleware.call(
-            'datastore.query', 'services.services', [('srv_service', '=', self._config.service)], {'get': True}
```

Comment:
Can we stop doing that on existing code as well? We are not following 80 char limit here

This is twicthing my OCD haha